testproject/urls.py

```diff
@@ -1,5 +1,5 @@
 try:
-    from django.urls import re_path, include
+    from django.urls import include, re_path
```

Comment:
I added those to support Django 1.x. As we want to drop the support, it should be removed. Here and there's one other place.

Reply:
[These are still supported in Django 3.x](https://docs.djangoproject.com/en/3.1/ref/urls/#re-path)

Reply:
Yes, these are still supported, but I added this try/except blog to support Django 1.x.

https://github.com/sunscrapers/djoser/blob/5e68bbbfda03a2d1c3cee414d491a8147a7669af/testproject/urls.py#L1-L5

commit https://github.com/sunscrapers/djoser/commit/a4778357fdc82ddcb1277585676cb8b77d2ab415

Reply:
Done.